eth/downloader/downloader.go

```diff
@@ -995,22 +1028,17 @@ func (d *Downloader) fetchBodies(from uint64) error {
 	)
 	err := d.fetchParts(errCancelBodyFetch, d.bodyCh, deliver, d.bodyWakeCh, expire,
```

Comment:
Here's something to maybe look at later. Massive signatures like this are becoming a red flag for me.